### PR TITLE
Add BdevLvolGetLvols method

### PR DESCRIPTION
**Pull request by derekbit**

### CODE REVIEW

**Review by innobead — APPROVED** [review]
LGTM

File: app/cmd/basic/bdev_lvol.go [hunk 1]
```diff
@@ -23,6 +23,7 @@ func BdevLvolCmd() cli.Command {
 			BdevLvolCloneCmd(),
 			BdevLvolDecoupleParentCmd(),
 			BdevLvolResizeCmd(),
+			BdevLvolListCmd(),
```
Comment (innobead) [inline app/cmd/basic/bdev_lvol.go:26]: From naming, it seems a little confusing between `BdevLvolGetCmd` and `BdevLvolListCmd`. What's the primary different for them when getting all lvols?

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM. Just one question.

File: pkg/spdk/client/basic.go [hunk 2]
```diff
@@ -133,6 +133,21 @@ func (c *Client) BdevLvolGetLvstore(lvsName, uuid string) (lvstoreInfoList []spd
...
 }
 
+// BdevLvolGetLvols receives either lvs_name or UUID.
+func (c *Client) BdevLvolGetLvols(lvsName, uuid string) (lvolInfoList []spdktypes.LvolInfo, err error) {
+	req := spdktypes.BdevLvolGetLvstoreRequest{
+		LvsName: lvsName,
+		UUID:    uuid,
+	}
+
+	cmdOutput, err := c.jsonCli.SendCommand("bdev_lvol_get_lvols", req)
```
Comment (shuo-wu) [inline pkg/spdk/client/basic.go:143]: I am thinking why not using `bdev_get_bdevs` with a filter? `bdev_get_bdevs` result provides more info.

File: app/cmd/basic/bdev_lvstore.go [hunk 3]
```diff
@@ -18,6 +18,7 @@ func BdevLvstoreCmd() cli.Command {
 			BdevLvstoreDeleteCmd(),
 			BdevLvstoreGetCmd(),
 			BdevLvstoreRenameCmd(),
+			BdevLvstoreGetLvolsCmd(),
```
Comment (shuo-wu) [inline app/cmd/basic/bdev_lvstore.go:21]: NIT: It's better to move this to `bdev_lvol.go`

Comment (shuo-wu) [reply app/cmd/basic/bdev_lvstore.go:21]: NIT: And you can name this to `BdevLvolGetByLvstoreCmd` instead.

**Review by innobead — APPROVED** [review]

File: pkg/spdk/client/basic.go (hunk 2 above)
Comment (derekbit) [reply pkg/spdk/client/basic.go:143]: `bdev_get_bdevs` lists all bdevs.
`bdev_lvol_get_lvols` can list lvols on a given lvstore. It is more convenient in some cases.

Comment (shuo-wu) [reply pkg/spdk/client/basic.go:143]: I knew. I mean, we can implement this API like what I do for RAID/Lvol/Nvme get functions.

File: app/cmd/basic/bdev_lvstore.go (hunk 3 above)
Comment (derekbit) [reply app/cmd/basic/bdev_lvstore.go:21]: Yeah, agree. I will add it with this https://github.com/longhorn/go-spdk-helper/pull/21#discussion_r1198769530. WDYT?

**Pull request merged**